### PR TITLE
Merge branch 'dev-6.0'

### PR DESCRIPTION
**Pull request by rdsantos18**
Driver SSD1963

### CODE REVIEW

File: display/SSD1963.c [hunk 1]
```diff
@@ -148,21 +148,20 @@ void ssd1963_init(void)
     LV_DRV_DELAY_MS(30);
 }
 
-
-void ssd1963_flush(int32_t x1, int32_t y1, int32_t x2, int32_t y2, const lv_color_t * color_p)
+void ssd1963_flush(lv_disp_drv_t * disp_drv, const lv_area_t * area, lv_color_t * color_p);
```
Comment (embeddedt) [inline display/SSD1963.c]: Syntax error at the end of this line. Can you compile test this?

**Comment by embeddedt** [comment]
I assume this fix has been tested on an SSD1963?

**Comment by rdsantos18** [comment]
> I assume this fix has been tested on an SSD1963?

tested hardware STM32F765 + SSD1963 24bit Color

**Comment by embeddedt** [comment]
Great! Thank you!

**Pull request merged**